common/lock.c

```diff
@@ -236,5 +241,11 @@ tdnflock tdnflockFree(tdnflock lock)
 		tdnflock_free(lock);
 	}
 
+	if (remove(TDNF_INSTANCE_LOCK_FILE))
+	{
+			pr_err("ERROR: Unable to remove lockfile(%s)\n",
```

Comment:
Make this a warning print? even though this file is not removed, there is no problem.